tsconfig.json

```diff
@@ -1,18 +1,26 @@
 {
 	"compilerOptions": {
-		"target": "es6",
-		"lib": [
-            "es6"
-        ],
 		"module": "commonjs",
-		"moduleResolution": "node",
+		"target": "ES2022",
 		"outDir": "out",
-		"resolveJsonModule": true
+		"lib": [
+			"ES2022"
+		],
+		"sourceMap": true,
+		"rootDir": "src",
+		"skipLibCheck": true,
+		"resolveJsonModule": true,
+		"strict": true, // true causes like million errors here
+		"forceConsistentCasingInFileNames": true
```

Comment:
When modifying that, you will need to check manually that it iss till working at runtime when installing the built vsix. there were issues with that in the past